modules/460-log-shipper/hooks/testdata/file-to-splunk/result.json

```diff
@@ -36,7 +36,22 @@
       "compression": "gzip",
       "default_token": "test-token",
       "endpoint": "192.168.1.1:9200",
-      "index": "{{ test }}"
+      "index": "{{ test }}",
+      "indexed_fields": [
+        "namespace",
+        "container",
+        "image",
+        "pod",
+        "node",
+        "pod_ip",
+        "stream",
+        "pod_owner",
+        "app"
```

Comment:
You have an "app" field here but it is absent in the splunk.go:
```
indexFields := []string{
```

Reply:
This is due to
```
  extraLabels:
    app: "{{ pod_labels.app }}"
```